client/state/reader/teams/reducer.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * External Dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	READER_TEAMS_REQUEST,
+	READER_TEAMS_REQUEST_FAILURE,
+	READER_TEAMS_REQUEST_SUCCESS,
+} from 'state/action-types';
+import { createReducer } from 'state/utils';
+
+export const items = createReducer( [], {
+	[ READER_TEAMS_REQUEST_SUCCESS ]: ( state, action ) => [ ...state, ...action.teams ],
```

Comment:
Teams are used in the Reader and are a property of individual users. It is how Automattic appears in the sidebar under "Following".  I don't think we support any teams other than `a8c` at the moment and don't have the historical context on what we were planning with it.

They aren't keyed by siteId or site.  Here is a sample response from the api for `read/teams`:
```javascript
{
	teams: [
		{
			title: 'Automattic',
			slug: 'a8c'
		}
	],
	number: 1
}
```

Although you got me thinking that I shouldn't be doing an append operation in the reducer, it should be a full replacement.